src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/GraphQL2RDFConfiguration.java

```diff
@@ -0,0 +1,63 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper;
+
+import java.util.Set;
+
+import se.liu.ida.hefquin.engine.federation.GraphQLEndpoint;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLFieldType;
+
+/**
+ * Used to represent and retrieve information about a virtual RDF view for a GraphQL endpoint
+ */
+public interface GraphQL2RDFConfiguration {
+
+    /**
+     * Verifies that @param uri that correponds to a class exists for @param endpoint
+     */
+    public boolean containsClassURI(final String uri, final GraphQLEndpoint endpoint);
+
+    /**
+     * Verifies that @param uri that corresponds to a property exists for @param endpoint
+     */
+    public boolean containsPropertyURI(final String uri, final GraphQLEndpoint endpoint);
+
+    /**
+     * @return a set of URI of all properties for @param className that links to a scalar value
+     * for @param endpoint
+     */
+    public Set<String> getPropertyURIs(final String className, final GraphQLFieldType fieldType, 
+        final GraphQLEndpoint endpoint);
+
+    /**
+     * @return the prefix used for properties
+     */
+    public String getPropertyPrefix();
+
+    /**
+     * @return the prefix used for classes
+     */
+    public String getClassPrefix();
+
+    /**
+     * @return the predefined namespace prefix for rdf
+     * https://www.w3.org/1999/02/22-rdf-syntax-ns#
+     */
+    public String getRDFPrefix();
```

Comment:
I'm still not sold on this method. Jena has a constant value with this URI, you can simply use `org.apache.jena.vocabulary.RDF.uri`, wherever you need this value. However, if you need to be able to customize the URI to indicate that an object belongs to a class (which in the default configuration is `rdf:type`) you can rename this method to `getClassMembershipURI` for instance, and make the default configuration return Jena's `RDF.type.getURI()`.